### ISSUE TITLE
[IDEA] Prevent the body from being scrolled when modal is open

### ISSUE THREAD

**Issue by ght** [Jermolene/TiddlyWiki5#5815]
It is really distracting to see the story river scrolling when being inside a modal and using the mouse wheel.

Scrolling should be contained in the modal. ~e.g. using the [`overscroll-behavior`](https://developer.mozilla.org/en-US/docs/Web/CSS/overscroll-behavior) CSS property.~ (`overscroll-behavior: contain` isn't sufficient as was discovered in #5816: If the modal fits without scrolling, there is no overscroll to contain, so the body will still be scrollable.)

(This issue was created out of @felixhayashi's PR #1558 from 2015 to fix this, all credit to him. Since `overscroll-behavior` was not yet available at the time, that fix had unwanted side effects, so this issue was created for tracking the problem)

**Comment by Jermolene** [comment]
Thanks @ght is the idea that the `overscroll-behavior` property is applied to the modal container?

**Comment by pmario** [comment]
The intention is, to apply it to the DIV `tc-modal-body` with `overscroll-behavior: contain` setting

**Comment by pmario** [comment]
I'll create a PR

**Comment by ght** [comment]
> Thanks @ght is the idea that the overscroll-behavior property is applied to the modal container?

That's my understanding; `overscroll-behavior` must be set on the scrollable element for which you want to prevent "scroll-chaining", i.e. the element where if you're at the bottom and scroll further down, you don't want to scroll the element's ancestors.

I use TiddlyWiki daily for ~1year now, but my usage is pretty basic, just creating/linking/transcluding tiddlers; I don't think I've seen a modal in TiddlyWiki before, and I've only started to dig through the source, so I'm afraid I can't help with specifics. I just opened this to help bring down the number of lingering open PRs.

---

> I'll create a PR
Thanks @pmario
